server/pulp/plugins/rsync/publish.py

```diff
@@ -258,6 +258,36 @@ def process_main(self):
             raise PulpCodedException(message=output)
 
 
+class UpdateLastPredistDateStep(PublishStep):
```

Comment:
so i was thinking whether it is the correct place for the Step. Since this step is used just by rpm plugin, i thought maybe it would be better to keep in plugins/pulp_rpm/plugins/distributors/?
i looked into docker plugin and i saw some docker specific steps located and subclassed in plugins/pulp_docker/plugins/distributors/publish_steps.py
What do you think of this? @dralley